Core/src/org/sleuthkit/autopsy/modules/photoreccarver/PhotoRecCarverFileIngestModule.java

```diff
@@ -256,6 +258,12 @@ public void startUp(IngestJobContext context) throws IngestModule.IngestModuleEx
             java.io.File newAuditFile = new java.io.File(Paths.get(outputDirPath.toString(), PHOTOREC_REPORT).toString()); //NON-NLS
```

Comment:
Please insert as cancellation check after writing the file to disk but before executing PhotoRec.